packages/firebase_ml/lib/src/model_management/firebase_model_manager.dart

```diff
@@ -30,24 +30,31 @@ class FirebaseModelManager {
   static final FirebaseModelManager instance = FirebaseModelManager._();
 
   /// Initiates the download of remoteModel if the download hasn't begun.
+  ///
+  /// If the model is already downloaded to the device, and there is no update, the task will immediately succeed.
+  /// If the model is already downloaded to the device, and there is update, a download for the updated version will be attempted.
```

Comment:
maybe also document what happens if updated version download fails?

Reply:
Flutter team tries to stay consistent with native implementations and documentation.
Current native (java/obj-c) documentation does not explicitly specify what happens if updated version download fails. 
I don't think we should do it either. 

@kroikie may provide more input on whether we should diverge from native documentation.

Reply:
I would feel free to add more context to documentation. Especially in cases where the native documentation may be less than detailed :)